__tests__/app.test.js

```diff
@@ -98,19 +98,68 @@ describe('3. Get api/articles/:article_id', () => {
     })
 
     describe('b. error handling', () => {
-        it('send a 404 status code when user inputs a bad path', () => {
+        it('send a 400 status code when user inputs a bad request', () => {
+            return request(app)
+                .get('/api/articles/article3')
+                .expect(400)
+                .then ((res) => {
+                    const {msg} = res.body;
+                    expect(msg).toBe('Bad Request')
+                })
+        })
+
+        it('send a 404 status code when user inputs a valid id but data does not exist', () => {
             return request(app)
-                .get('/api/artiales/3')
+                .get('/api/articles/99999')
                 .expect(404)
                 .then ((res) => {
                     const {msg} = res.body;
                     expect(msg).toBe('Not Found')
                 })
         })
+    })
+})
+
+describe('4. GET /api/articles/:article_id/comments', () => {
+    describe('a. status 200 & data', () => {
+        it('returns 200 status code & an array back to the user where the data', () => {
+            return request(app)
+            .get('/api/articles/3/comments')
+            .expect(200)
+            .then((res) => {
+                const articles = res.body;
+                expect(articles).toBeInstanceOf(Array);

```

Comment:
You are using .forEach() array method, so is it necessary to check for array here?